ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -238,8 +251,15 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
             );
         }
 
+        const counts = this.countsByCompleted();
+
         return (
             <>
+                {(counts.complete > 0 || counts.incomplete > 100) && (
+                    <CostOptimisationNudge name='workflow-list'>
+                        You have at least {counts.incomplete} incomplete, and {counts.complete} complete workflows. Reducing these amounts will reduce your costs.
```

Comment:
I'm aware this may show zeros: "You have at least 0 incomplete", but it is simple

Reply:
Similar comment here to linking to the cost optimization docs.

Reply:
as previous comment - already done